go.mod

```diff
@@ -3,12 +3,14 @@ module github.com/redhat-developer/gitops-operator
 go 1.13
 
 require (
-	github.com/openshift/api v3.9.0+incompatible
+	github.com/argoproj-labs/argocd-operator v0.0.13
+	github.com/go-logr/logr v0.1.0
+	github.com/google/go-cmp v0.4.0
+	github.com/openshift/api v3.9.1-0.20190916204813-cdbe64fb0c91+incompatible
 	github.com/operator-framework/operator-sdk v0.18.2
-	github.com/prometheus/common v0.9.1
```

Comment:
Can we remote the following under `replace` now that `prometheus` is removed?

```
k8s.io/client-go => k8s.io/client-go v0.18.2 // Required by prometheus-operator
```

Reply:
@wtam2018 Tried removing this replace. It is throwing errors. It may be dependent on the operator-sdk version